Cargo.toml

```diff
@@ -55,6 +55,7 @@ workspace = true
 
 [dependencies]
 vello_encoding = { workspace = true }
+vello_shaders = { workspace = true }
```

Comment:
Do we still need the `compile` feature to be enabled even if we're not hot reloading?

I guess it doesn't really matter, because we share the Naga version with `wgpu`

Reply:
That's a good point. Even though the features match, I think there is merit to excluding the feature when it's not needed. Though I generally tend to rely on linkers being smart about excluding unused code from binaries (which is desirable in the wasm build case) I went ahead and made "compile" an optional feature, which feels right to me.